staging_service/app.py

```diff
@@ -99,6 +99,7 @@ async def file_exists(request: web.Request):
             show_hidden = False
     except KeyError as no_query:
         show_hidden = False
+    # this scans the entire directory recursively just to see if one file exists... why?
```

Comment:
I assume there's some reason for doing so much work for something that seems trivial... if anyone knows what it is it'd be good to document is because naively this seems crazy

Reply:
Have you looked in other repos to see if they call this endpoint?

Reply:
https://kbase-jira.atlassian.net/browse/SCT-276